src/Raven.Client/Documents/Operations/ETL/Queue/QueueBrokerType.cs

```diff
@@ -4,5 +4,6 @@ public enum QueueBrokerType
 {
     None,
     Kafka,
-    RabbitMq
+    RabbitMq,
+    AzureQueueStorage
```

Comment:
We share this enum with Queue Sink feature, right? We need to ensure we prevent from using `AzureQueueStorage` together with Queue sinks by throwing appropriate exception

Reply:
Was it done?